src/App.tsx

```diff
@@ -50,9 +52,11 @@ export const App = (): JSX.Element => {
 
   return (
     <Routes>
-      <Route path={buildMainPath()} element={<ItemPage />} />
-      <Route path={HOME_PATH} element={<HomePageWithAuthorization />} />
-      <Route element={<Navigate to={HOME_PATH} />} />
+      <Route element={<PageWrapper />}>
+        <Route path={buildMainPath()} element={<ItemPage />} />
+        <Route path={HOME_PATH} element={<HomePageWithAuthorization />} />
+        <Route path="*" element={<Navigate to={HOME_PATH} />} />
```

Comment:
If no routes path are matched, it redirect the user to the home page ? Shouldn't be clearer to let the user on the current page and show him a 404 error ?